Rakefile

```diff
@@ -1,37 +1,50 @@
 require_relative 'lib/pdksync'
 require 'github_changelog_generator/task'
 
-
-desc 'Run full pdksync process, clone repo, pdk update, create pr.'
+desc 'Run full pdksync process, clone repository, pdk update, create pr.'
 task :pdksync do
   PdkSync::run_pdksync
 end
 
-desc 'Clone managed modules'
-task :clone_managed_modules do
-  PdkSync::main(steps: [:clone])
-end
+namespace :pdk do 
+  desc 'Runs PDK convert against modules'
+  task :pdk_convert do
+    PdkSync::main(steps: [:pdk_convert])
```

Comment:
This is probably extended functionality, but should we be able to take arguments in for the sake of 'template_url'? Would be useful for people to be able to determine their own template.